userspace/libraries/libutils/ArgParse.h

```diff
@@ -110,7 +121,7 @@ class ArgParse
 
     void show_help_if_no_option_given() { _show_help_if_no_option_given = true; }
 
-    ArgParse()
+    ArgParse(const ArgParseVersion &v = {0, 0, 1}) : _version(v)
```

Comment:
Can't we use the version number pass to us by the build system?

Reply:
Yes. The idea was that each utility can still be versioned individually though